applications/main/archive/helpers/archive_browser.h

```diff
@@ -24,25 +24,16 @@ static const char* tab_default_paths[] = {
 };
 
 static const char* known_ext[] = {
-    [ArchiveFileTypeIButton] = ".ibtn",
-    [ArchiveFileTypeNFC] = ".nfc",
-    [ArchiveFileTypeSubGhz] = ".sub",
-    [ArchiveFileTypeLFRFID] = ".rfid",
-    [ArchiveFileTypeInfrared] = ".ir",
-    [ArchiveFileTypeSubghzPlaylist] = ".txt",
-    [ArchiveFileTypeSubghzRemote] = ".txt",
-    [ArchiveFileTypeInfraredRemote] = ".txt",
-    [ArchiveFileTypeBadKb] = ".txt",
-    [ArchiveFileTypeWAV] = ".wav",
-    [ArchiveFileTypeU2f] = "?",
-    [ArchiveFileTypeApplication] = ".fap",
-    [ArchiveFileTypeJS] = ".js",
-    [ArchiveFileTypeSearch] = "*",
-    [ArchiveFileTypeUpdateManifest] = ".fuf",
-    [ArchiveFileTypeDiskImage] = ".img",
-    [ArchiveFileTypeFolder] = "?",
-    [ArchiveFileTypeUnknown] = "*",
-    [ArchiveFileTypeAppOrJs] = ".fap|.js",
+    [ArchiveFileTypeIButton] = ".ibtn",     [ArchiveFileTypeNFC] = ".nfc",
+    [ArchiveFileTypeSubGhz] = ".sub",       [ArchiveFileTypeLFRFID] = ".rfid",
+    [ArchiveFileTypeInfrared] = ".ir",      [ArchiveFileTypeSubghzPlaylist] = ".txt",
+    [ArchiveFileTypeSubghzRemote] = ".txt", [ArchiveFileTypeInfraredRemote] = ".txt",
+    [ArchiveFileTypeBadKb] = ".txt",        [ArchiveFileTypeWAV] = ".wav",
+    [ArchiveFileTypeMag] = ".mag",          [ArchiveFileTypeU2f] = "?",
+    [ArchiveFileTypeApplication] = ".fap",  [ArchiveFileTypeJS] = ".js",
+    [ArchiveFileTypeSearch] = "*",          [ArchiveFileTypeUpdateManifest] = ".fuf",
+    [ArchiveFileTypeDiskImage] = ".img",    [ArchiveFileTypeFolder] = "?",
+    [ArchiveFileTypeUnknown] = "*",         [ArchiveFileTypeAppOrJs] = ".fap|.js",
```

Comment:
I hate when it formats the arrays like this, will make it so annoying to merge future changes too. I don't think there's a consistent way to make it not do that, I'll try to fiddle with it a bit

Reply:
Drives me up a wall too; it made one of the lists in my transit card parser practically unreadable. If there's an easy way to disable (and that won't cause issues a la failing lint on other FWs) I'd love to.

Reply:
found it )